images/authoring/tomcat/docker-entrypoint.sh

```diff
@@ -23,6 +23,18 @@ chown_dir() {
   fi
 }
 
+host_keyscan() {
+    if [ ! -d $CRAFTER_SSH_CONFIG ]; then
+        mkdir -p $CRAFTER_SSH_CONFIG
+        chown_dir "$CRAFTER_SSH_CONFIG"
+    fi
+    declare -a DOMAINS=("bitbucket.com" "gitlab.com" "github.com")
```

Comment:
Can you maybe make this a env variable that can be overwritten?

Reply:
Updated to have env variable `SSH_KEYSCAN_DOMAINS` with a default value if not set and separated by `,`.

Example to set:

```
export SSH_KEYSCAN_DOMAINS=bitbucket.com,gitlab.com,github.com
```